media/js/firefox/feedback-ga-tracking.js

```diff
@@ -6,47 +6,17 @@
     'use strict';
 
     function trackGenericLink(action, href, callback) {
-        if (typeof callback === 'function') {
-            gaTrack(['_trackEvent', 'Feedback Interactions', action, href], callback);
-        } else {
-            gaTrack(['_trackEvent', 'Feedback Interactions', action, href]);
-        }
+
     }
 
     // track clicks in the secondary CTAs
     $('.action-secondary').on('click', 'a', function(e) {
-        var name = $(this).data('name');
-        var href = this.href;
-        var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
-        var callback;
-
-        if (newTab) {
-            trackGenericLink('CTA click', name);
-        } else {
-            e.preventDefault();
-            callback = function() {
-                window.location = href;
-            };
-            trackGenericLink('CTA click', name, callback);
-        }
+
     });
 
     // track clicks in the SUMO links
     $('.support-links').on('click', 'a', function(e) {
-        var label = $(this).text();
-        var href = this.href;
-        var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
-        var callback;
-
-        if (newTab) {
-            trackGenericLink('SUMO click', label);
-        } else {
-            e.preventDefault();
-            callback = function() {
-                window.location = href;
-            };
-            trackGenericLink('SUMO click', label, callback);
-        }
+
```

Comment:
Looks like we can delete this file and all references now that it has no purpose.